Makefile.release

```diff
@@ -44,7 +44,7 @@ EMPTY:=
 SPACE:=$(EMPTY) $(EMPTY)
 COMMA:=$(EMPTY),$(EMPTY)
 
-ifeq (, $(shell command -v curl))
```

Comment:
Why moving back to which? `command -v` is posix https://pubs.opengroup.org/onlinepubs/009604499/utilities/command.html https://github.com/koalaman/shellcheck/wiki/SC2230

Reply:
On Ubuntu 20.04 with `command` I noticed the following errors:
```
$ make -f Makefile.release  tar
make: command: Command not found
Makefile.release:48: *** "No curl in $PATH, please install".  Stop.
```

Reply:
Okay, I don't have a ubuntu machine to test here. Then yes revert back to `which curl` as @yongtang , I was hoping that we could avoid relying on which.